SECURITY.md

```diff
@@ -0,0 +1,57 @@
+# Extensions Security Guidelines
+
+OpenSearch's support for extensions allows for taking already powerful use cases and expanding on them, this creates a larger surface area for misuse, vunerabilities, and malicious interactions.  This document outlines several areas for enhancements, features, and practices to incorperate into extensions for OpenSearch
+
+To keep concepts consistant, this document is using terminology from [NIST Glossary](https://csrc.nist.gov/glossary).
+
+Additional terms:
+* **Plugin** - reference to the existing functionality to extend OpenSearch functionality.
+* **Extension** - reference to the in development functionality to extend OpenSearch.
+
+## Host security
+
+The Java Security Manager is the mechanism for ensuring plugins are limited in what they can do to the host operation system resources (cpu/disk/memory/network/...).  As there are limitations and its deprecated with removal scheduled in the next release of the JVM.
```

Comment:
I'm unclear about "is the mechanism" vs. "it's deprecated".    If it's on its way out this should be much more past-tense focused, e.g., "While the Java Security Manager has been.... " and add on "Additional measures are needed to protect system resources."

Reply:
Rewording to make this clearer - let me know what you think of the revision